docs/performance/inserts/testing.txt

```diff
@@ -0,0 +1,96 @@
+.. _testing_inserts_performance:
+
+===========================
+Testing Inserts Performance
+===========================
+
+Every setup is different. The best way to tune your particular setup is to do
+performance testing.
```

Comment:
I'm wondering if it would make sense to point out that it's reasonable to test against a single node first, since inserts scale pretty much linearly.

Reply:
done